CompilerSource/ipc/Sound.fbs

```diff
@@ -0,0 +1,27 @@
+
+enum SoundKind : byte { NORMAL, BACKGROUND_MUSIC, THREE_DIMENSIONAL, MULTIMEDIA_PLAYER }

```

Comment:
I don't know if we want to start dropping some of this stuff from legacy GM that ENIGMA isn't using. This is one particular case, but we are also not including the new properties from GMS to specify the encoding and such. We also don't have code for that engine side. We had discussed changing this to offer some encoding options to make games smaller. There's also a "Type" in GMX which is completely distinct from "Kind" but both are serialized. Type has the options "Mono," "Stereo," or "3D." Thoughts?